.gitmodules

```diff
@@ -1,6 +1,6 @@
-[submodule "lib/forge-std"]
-	path = lib/forge-std
+[submodule "forge/forge-std"]
+	path = forge/forge-std
 	url = https://github.com/foundry-rs/forge-std
-[submodule "lib/murky"]
-	path = lib/murky
+[submodule "forge/murky"]
+	path = forge/murky
```

Comment:
Same as in https://github.com/0xPolygon/core-contracts/pull/202/files#r1172462574